src/docfx/build/metadata/CommitsTimeItem.cs

```diff
@@ -4,11 +4,14 @@
 using System;
 using Newtonsoft.Json;
 
-internal class CommitsTimeItem
+namespace Microsoft.Docs.Build
```

Comment:
is there a run to valid?

Reply:
I cannot find an existing stylecop rule that enforce that